sdk/core/azure-core/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Breaking Changes
 
 - `Azure::Core::Http::Url::AppendPath` now does not encode the input by default.
+- `azure.hpp` renamed to `strings.hpp`
+- `Azure::Core::Details::LocaleInvariantCaseInsensitiveEqual` moved to `Azure::Core::Strings::LocaleInvariantCaseInsensitiveEqual`
```

Comment:
Same here, not a breaking change.

Reply:
Ashon is correct, It's technically not a breaking change b/c it was in details namespace.